libs/entry-components/styles/modules/_default-theme.scss

```diff
@@ -45,6 +45,9 @@ $theme: (
 		inputs: (
 			background: null
 		),
+		paginators: (
```

Comment:
Since paginators are inheriting background from inputs by default, maybe it should be implemented as submap for inputs instead of separate entity?

Reply:
I'm also bit unsure whether this paginator thing is mature enough to end in theme. It's more logical for all fields to have the same color and we have only one exception for this logic in our projects so far.